src/site/_includes/components/EventTable.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// const path = require('path');
+const {html} = require('common-tags');
+// const prettyDate = require('../../_filters/pretty-date');
+// const stripLanguage = require('../../_filters/strip-language');
+// const md = require('../../_filters/md');
+// const constants = require('../../_utils/constants');
+// const getSrcsetRange = require('../../_utils/get-srcset-range');
+// const postTags = require('../../_data/postTags');
+
+const Author = require('./Author');
+
+module.exports = (collections) => {
+  const schedule = collections.eventSchedule;
+
+  // TODO(MichaelSolati): "Author" needs a post but doesn't use it, we pass an empty object.
```

Comment:
As an alternative you could use the `w-author` HTML / classes directly. The shortcode is syntactic sugar but if you're having to pass it bogus arguments that feels off.

Reply:
The Author shortcode is actually quite convenient here, I really like it.

This comment (cc @MichaelSolati) was mostly observing that I think the Author shortcode has `post` left over from an older time, it's just not used.

Another bit of feedback might be that it could be nice to automatically (yet again I'm confused about Eleventy's data) load the "author" information from the `id`. Since I've had to do a bunch of work to load it up correctly.